ui/src/app/pages/access/access-renewal.tsx

```diff
@@ -133,37 +133,24 @@ const syncAndReloadTraining = fp.flow(
 });
 
 // Helper Functions
-// The module has already expired
-export const hasExpired = (expiration: number): boolean =>
-  !!expiration && getWholeDaysFromNow(expiration) < 0;
-
-// The module can either be expired or is expiring
-export const isExpiring = (expiration: number): boolean =>
-  expiration
-    ? getWholeDaysFromNow(expiration) <=
-      serverConfigStore.get().config.accessRenewalLookback
-    : false;
-
-const isModuleExpiring = (status: AccessModuleStatus): boolean =>
-  isExpiring(status?.expirationEpochMillis);
-
-export const isExpiringNotBypassed = (moduleStatus: AccessModuleStatus) => {
```

Comment:
the only usage is just below here, which I inlined to avoid the confusing similarity of these names